### PR TITLE
Change to css timeline

### PR DESCRIPTION
**Pull request by hillairet**
Switched to a much better timeline using only CSS for rendering.
Javascript still used to load the data into HTML.

### CODE REVIEW

File: data.js [hunk 1]
```diff
@@ -1,101 +1,106 @@
...
+          {"date": "1918-08-08", "title": "Canada is victorious at the Battle of Amiens which is considered as \"the Black day of the German Army\""},
+          {"date": "1918-11-11", "title": "World Ward II Armistice", "description": ["60,000 Canadians were killed, 170,000 wounded.", "November 11 eventually becomes Remembrance day."]},
+          {"date": "1939-09-01", "title": "Beginning of World War II", "description": "The Royal Canadian Navy (RCN) saw its finest hour in the Battle of the Atlantic protecting convoys of merchant ships against German submarines"},
+          {"date": "1940-07-10", "title": "The Royal Canadian Air Force (RCAF) takes part in the Battle of England", "description": "The RCAF provides a high portion of Commonwealth aircrew in bombers and fighter planes over Europe."},
+          {"date": "1944-06-06", "title": "D-Day, invasion of Normandy", "description": "15,000 Canadian troops captune Juno beach"},
```
Comment (rahulpatidar0191) [inline data.js]:
```suggestion
          {"date": "1944-06-06", "title": "D-Day, invasion of Normandy", "description": "15,000 Canadian troops captured Juno beach"},
```

File: data.js [hunk 2]
```diff
@@ -1,101 +1,106 @@
...
+          {"date": "1945-09-02", "title": "End of World War II", "description": "More than 1 million Canadians and Newfoundlanders served in Wolrd War II"},
```
Comment (rahulpatidar0191) [inline data.js]:
```suggestion
          {"date": "1945-09-02", "title": "End of World War II", "description": "More than 1 million Canadians and Newfoundlanders served in World War II"},
```

File: data.js [hunk 3]
```diff
@@ -1,101 +1,106 @@
...
+          {"date": "1793-07-09", "title": "Upper Canada, led by Lieutenant Governor John Graves Simcoe, becomes the first province in the Empire to move towards abolition"},
+          {"date": "1807-03-25", "title": "The British Parliament prohibits buying and selling of slaves"},
+          {"date": "1840-07-23", "title": "Upper and Lower Canada are united into the Province of Canada.", "description": "Sir Louis-Hippolyte Lafontaine, Robert Baldwin and Joseph Howe work with British governors towards responsible government"},
+          {"date": "1848-01-25", "title": "Nova Scotia is the first North American colony to attain full responsible government"},
+          {"date": "1849-02-27", "title": "The governor of United Canada, Lorg Egin, introduces a responsible government", "description": "Lafontaine, a champion of democracy and French language rights, becomes the first leader of a responsible government in the Canadas"},
```
Comment (rahulpatidar0191) [inline data.js]:
```suggestion
          {"date": "1849-02-27", "title": "The governor of United Canada, Lord Elgin, introduces a responsible government", "description": "Lafontaine, a champion of democracy and French language rights, becomes the first leader of a responsible government in the Canadas"},
```

File: data.js [hunk 4]
```diff
@@ -1,101 +1,106 @@
...
+          {"date": "1864-09-01", "title": "The Charlottetown Conference marks the beginning of the Confederation of Canada which is completed in 1867.", "description": ["Representatives from Nova Scotia, New Brunswick and Province of Canada form a new country.", "Federal and provincial governments are created.", "Province of Canada is split into Ontario & Quebec", "Sir George-Etienne Cartier, a key architect of Confederation from Quebec, leads Quebec into confederation."]},
+          {"date": "1867-07-01", "title": "The Birtish North America Act is enacted", "description": ["The Dominion of Canada is born, term used for 100 years.", "July 1st is referred to as \"Dominion Day\" till 1982"]},
```
Comment (rahulpatidar0191) [inline data.js]:
```suggestion
          {"date": "1867-07-01", "title": "The British North America Act is enacted", "description": ["The Dominion of Canada is born, term used for 100 years.", "July 1st is referred to as \"Dominion Day\" till 1982"]},
```

File: data.js [hunk 5]
```diff
@@ -1,101 +1,106 @@
...
+          {"date": "1870-05-12", "title": "Canada establishes the new province of Manitoba"},
+          {"date": "1871-07-20", "title": "BC joins Canada after Ottawa promises to build a railway to the West Coast"},
+          {"date": "1873-05-23", "title": "John A. Macdonald establishes the North West Mounted Police (NWMP) to pacify the West", "description": "One of Canada's most colourful heroes, General Sir Sam Steele, comes from the ranks of the mounties"},
+          {"date": "1873-07-01", "title": "Prince Edward Island joins Canada"},
+          {"date": "1880-09-01", "title": "Transfer of the Artic Islands from Britain to the North West Territories"},
```
Comment (rahulpatidar0191) [inline data.js]:
```suggestion
          {"date": "1880-09-01", "title": "Transfer of the Arctic Islands from Britain to the North West Territories"},
```

File: data.js [hunk 6]
```diff
@@ -1,101 +1,106 @@
...
+          {"date": "1792-01-15", "title": "Black Nova Scotians who were given poor lang move to Sierra Leone to establish Freetown", "emoji": "🇸🇱"},
+          {"date": "1793-01-21", "title": "The first Assembly of Lower Canada debates whether to use both French and English", "emoji": "🗣️"},
+          {"date": "1832-03-30", "title": "The Montreal Stock Exchange opens", "emoji": "💱"},
+          {"date": "1839-02-11", "title": "Lord Durham, an English reformer, recommends that Upper and Lower Canada be merged and given responsible government", "emoji": "🔗"},
+          {"date": "1850-01-01", "title": "During the 1850s, the maple lead starts appearing on Canadian uniforms & insignia", "emoji": "🍁"},
+          {"date": "1867-07-01", "title": "Sir John Alexander MacDonald is Canada's 1st Prime Minister and a father of Confederation.", "description": ["January 11th is Sir John A. Macdonald day.", "His portrait is on the $10 bill."], "emoji": "1️⃣"},
+          {"date": "1870-09-02", "title": "Louis Riel flees to the US", "emoji": "🏃🏻‍♂️"},
+          {"date": "1885-11-07", "title": "The Canadian Pacific Ralway's \"Ribbons of steel\" is completed", "emoji": "🛤️"},
```
Comment (rahulpatidar0191) [inline data.js]:
```suggestion
          {"date": "1885-11-07", "title": "The Canadian Pacific Railway's \"Ribbons of steel\" is completed", "emoji": "🛤️"},
```

File: data.js [hunk 7]
```diff
@@ -1,101 +1,106 @@
...
+          {"date": "1885-11-16", "title": "Riel is executed after he was found guilty of treason despite the opposition of Quebec", "emoji": "💀"},
+          {"date": "1896-07-11", "title": "Wilfrid Laurier becomes the 1st French Canadian Prime Minister", "description": "He holds the longest unbroken term of office with 15 years", "emoji": "1️⃣"},
+          {"date": "1921-12-06", "title": "Agnes Macphail, a farmer and teacher, becomes the 1st woman MP", "emoji": "🗳️"},
+          {"date": "1929-10-29", "title": "The stock market crashes leading to the Great Recession.", "emoji": "📉"},
+          {"date": "1933-06-01", "title": "Unemployment reaches 27% an many businesses are wipedout.", "emoji": "📉"},
```
Comment (rahulpatidar0191) [inline data.js]:
```suggestion
          {"date": "1933-06-01", "title": "Unemployment reaches 27% and many businesses are wiped out.", "emoji": "📉"},
```

**Review by rahulpatidar0191 — CHANGES_REQUESTED** [review]
Some typos

**Comment by hillairet** [comment]
@rahulpatidar0191 Good eye for typos! Thanks!
Any feedback on the overall look?

**Comment by rahulpatidar0191** [comment]
The overall look is good. Grouping them with emoji is a clever idea. 

One very minor thing I noticed is the single points are missing a bullet. For eg: John Cabot is missing a bullet but the next point has it. Again, minor thing 

image

**Comment by hillairet** [comment]
> The overall look is good. Grouping them with emoji is a clever idea.
> 
> One very minor thing I noticed is the single points are missing a bullet. For eg: John Cabot is missing a bullet but the next point has it. Again, minor thing
> 
> image

It's actually by design!  :sweat_smile: 
I thought it'd be better to separate between one description and multiple but clearly it looks off since you thought it was a mistake.
I'll put a bullet point every time to make it consistent.

**Review by rahulpatidar0191 — APPROVED** [review]

**Pull request merged**